### PR TITLE
created feature B

### PR DESCRIPTION
**Pull request by huskybinary**
Hey bro...see I created feature B again dude....can you check this out

### CODE REVIEW

File: code.txt [hunk 1]
```diff
@@ -5,4 +5,6 @@ Now I have implemented feature A here...I really hope the maintainer likes it.
 
 
 Now another new feature by the maintainer...Hehe
-Ok I commit again...hehe
```
Comment (euchengc) [inline code.txt:8]: Why do you delete my code dafuq

File: code.txt [hunk 2]
```diff
@@ -5,4 +5,6 @@ Now I have implemented feature A here...I really hope the maintainer likes it.
...
\ No newline at end of file
+Ok I commit again...hehe
```
Comment (euchengc) [inline code.txt:8]:
```suggestion
Ok I commit again...hehe
```

Huh... so you readded it back?

**Review by euchengc — CHANGES_REQUESTED** [review]
This is kinda weird

File: code.txt (hunk 1 above)
Comment (euchengc) [reply code.txt:8]: nvm its ok

File: code.txt (hunk 2 above)
Comment (huskybinary) [reply code.txt:8]: I agree. I'll add another he.

File: code.txt (hunk 1 above)
Comment (huskybinary) [inline code.txt:8]: Who is the maintainer tho

**Review by huskybinary — COMMENTED** [review]
Please answer my question dumbass

File: code.txt [hunk 3]
```diff
@@ -5,4 +5,6 @@ Now I have implemented feature A here...I really hope the maintainer likes it.
...
+
+I'm adding feature B!!!!!
```
Comment (huskybinary) [inline code.txt]: I really like my code

Comment (huskybinary) [inline code.txt:10]:
```suggestion
Ok I commit again...hehe

I'm adding feature B!!!!!
```

What about feature C???

File: code.txt (hunk 1 above)
Comment (huskybinary) [reply code.txt:8]: The maintainer is me dumbass

File: code.txt (hunk 3 above)
Comment (huskybinary) [reply code.txt]: Yeah what about it. Go do your job

File: code.txt [hunk 4]
```diff
@@ -5,4 +5,6 @@ Now I have implemented feature A here...I really hope the maintainer likes it.
 
 
 Now another new feature by the maintainer...Hehe
-Ok I commit again...hehe
\ No newline at end of file
+Ok I commit again...hehehe
```
Comment (huskybinary) [inline code.txt:8]:
```suggestion
Ok I commit again now...hehehe
```

File: code.txt [hunk 5]
```diff
@@ -5,4 +5,6 @@ Now I have implemented feature A here...I really hope the maintainer likes it.
...
+
+I'm adding feature B!!!!!
```
Comment (huskybinary) [inline code.txt:10]:
```suggestion
I'm adding feature B!!!!! and also feature C soon!
```

**Review by euchengc — COMMENTED** [review]
Ok it's ok

**Review by euchengc — APPROVED** [review]

**Pull request merged**